environment.yml

```diff
@@ -1,7 +1,7 @@
 name: rpxdock
 channels:
  - conda-forge
- - https://username:password@conda.graylab.jhu.edu
+ - https://pose:foldtree@conda.graylab.jhu.edu
```

Comment:
revert this line, pyrosetta is still not open source so the username and password can't be open :$